### PR TITLE
Add info about discussion notifications to ICR guide

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3859](https://openedx.atlassian.net/browse/DOC-3859)

Add info about how to enable discussion notifications to _Installing... the Open edX Platform_

### Date Needed (optional)

ASAP

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @iloveagent57 
- [ ] Doc team review: @edx/doc 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/install_operations/source/configuration/enable_discussion_notifications.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. _Enable Discussion Notifications:
+
+##########################################
+Enable Discussion Notifications
+##########################################
+
+You can set up notifications that learners receive the first time that anyone
+adds a response to a discussion post that they have made.
+
+For more information, see :ref:`opencoursestaff:ADD_REFERENCE` and
```
Comment (catong) [inline en_us/install_operations/source/configuration/enable_discussion_notifications.rst]: These references need to be added

File: en_us/install_operations/source/configuration/enable_discussion_notifications.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.. _Enable Discussion Notifications:
+
+##########################################
+Enable Discussion Notifications
```
Comment (catong) [inline en_us/install_operations/source/configuration/enable_discussion_notifications.rst]: Top level and next level heading are identical. Our style is to use gerunds for the conceptual topic, so I'd recommend changing the top level heading to "Enabling" - will also make it consistent with other ICR headings

**Review by catong — CHANGES_REQUESTED** [review]

**Review by iloveagent57 — APPROVED** [review]

File: en_us/install_operations/source/configuration/enable_discussion_notifications.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
+.. _Enabling Discussion Notifications:
```
Comment (srpearce) [inline en_us/install_operations/source/configuration/enable_discussion_notifications.rst:1]: @iloveagent57 – would you remind reviewing this topic again? I forgot the info about ACE and a third party email client in the earlier version.

**Comment by srpearce** [comment]
@catong Would you mind looking over this again? I think the install guide topic might be a bit awkward so would love a second opinion.

File: en_us/install_operations/source/configuration/enable_discussion_notifications.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+##########################################
+Enabling Discussion Notifications
+##########################################
+
+You can set up notifications that learners receive the first time that anyone
+adds a response to a discussion post that they have made.
+
+For more information, including the text of the discussion notification
+message, see :ref:`opencoursestaff:Automatic Email` and
```
Comment (srpearce) [inline en_us/install_operations/source/configuration/enable_discussion_notifications.rst:11]: @catong This link seems to be broken, but I can't figure out why. Any ideas?

Comment (catong) [reply en_us/install_operations/source/configuration/enable_discussion_notifications.rst:11]: @srpearce If the "Automatic Email" anchor in the other file is new to this PR, the ref won't be found until you publish that guide and the target exists to be found in RTD. Could that be it?

File: en_us/course_authors/source/manage_live_course/automatic_email.rst [hunk 5]
```diff
@@ -1 +1,199 @@
...
+   - Self-paced
+   - Every 7 days after enrollment
+   - Weekly message that lists 3-5 highlights for the
+     next course "week", or section. Is not sent if the course team has not
+     added highlights for the section in Studio. For more information, see
+     :ref:`Set Section Highlights for Weekly Course Highlight Messages`.
+ * - :ref:`Discussion Notification`
+   - Instructor-paced and self-paced
+   - Unspecified
+   - Notifications that a user has responded to a post on a discussion.
```
Comment (catong) [inline en_us/course_authors/source/manage_live_course/automatic_email.rst]: Might be helpful to link "discussion" to the section on discussions.

File: en_us/install_operations/source/configuration/enable_discussion_notifications.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+:ref:`openlearners:Receiving Discussion Notifications`.
+
+.. contents::
+   :depth: 1
+   :local:
+
+.. _Obtain Discussion Notification Requirements:
+
+*******************************************
+Obtain Discussion Notification Requirements
```
Comment (catong) [inline en_us/install_operations/source/configuration/enable_discussion_notifications.rst]: A heading such as "Requirements for Enabling Discussion Notifications" would make it more clear what's being communicated here. I had to read several times before I really figured out what the topic was for. "Obtain..." makes more sense for a procedure; this isn't a procedure.

**Review by catong — CHANGES_REQUESTED** [review]
@srpearce A couple of additional suggestions, as requested, then 👍

**Review by iloveagent57 — APPROVED** [review]

**Pull request merged**